programming/motors.md

```diff
@@ -6,77 +6,82 @@ title: Motor Board API
 Motor Board API
 ===============
 
-The `motor_board` object is used to control a collection of Motor Boards.
+The kit can control multiple motors simultaneously
+Each Motor Board can control two motors.
+See the [Motor Board](/docs/kit/motor_board) hardware page for more details.
 
-When a single Motor Board is connected to your robot, you can control it
-using the `motor_board` object.
+
+[Accessing the Motor Board](#access_motor_power) {#access_motor_power}
+----------------------------------------------------------------------
+
+If there is exactly one Motor Board connected to your robot, it can be accessed using the `motor_board` property of the `Robot` object.
 
 ~~~~~ python
-R.motor_board.something...
+from sr.robot3 import *
+robot = Robot()
+
+my_motor_board = robot.motor_board
 ~~~~~
 
-The serial number of each detected Motor Board is printed to the log when your robot starts.
+If you have more than one Motor Board attached, you need to specify which one you want to control.
+This is done using the serial number of the board.
+
+The serial number is physically written on each board and the serial of any connected board will be printed to your log when the robot starts.
 It will look something like this:
 
 ~~~~~ not-code
-sr.robot3.robot INFO - Found Student Robotics Motor Board v4 - srABC1
+sr.robot3.robot - INFO - Found MotorBoard, serial: srABC1
 ~~~~~
 
-If you have more than one Motor Board attached, you need to specify which one you want to control. This is done using the serial number of the board. For example: if you had a board that was detected as "srABC1",
-you could do this instead:
+You can then access the boards like this:
 
 ~~~~~ python
-R.motor_boards["srABC1"].something...
+from sr.robot3 import *
+robot = Robot()
+
+my_motor_board = robot.motor_boards["srABC1"]
+my_other_motor_board = robot.motor_boards["srXYZ1"]
 ~~~~~
 
 <div class="warning" markdown="1">
-  When you have more than one Motor board connected to your kit,
-  you must use `R.motor_boards` and index by serial number. This is so
-  that the kit knows which Motor Board you want to control.
+  When you have more than one Motor Board connected to your kit, you must use `robot.motor_boards` and index by serial number.
```

Comment:
Nit: Be specific that the issue is not using `robot.motor_board` vs needing to use `robot.motor_boards`.

```suggestion
  When you have more than one Motor Board connected to your kit, you can't use `robot.motor_board`.
```

Reply:
a40db43